test/adapters/integration.rb

```diff
@@ -59,7 +59,8 @@ def test_no_parallel_support
     module Compression
       def test_GET_handles_compression
         res = get('echo_header', :name => 'accept-encoding')
-        assert_match(/gzip;.+\bdeflate\b/, res.body)
```

Comment:
Since `httpclient` sends `Accept-Encoding: gzip,deflate` with a `,` instead of an `;`. This is valid as  described by http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.3 .